jubakit/wrapper.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin
+from .classifier import Classifier, Config, Dataset
+
+
+class BaseJubatusClassifier(BaseEstimator, ClassifierMixin):
+  """
+  scikit-learn Wrapper for Jubatus Classifiers.
+  """
+  
+  def __init__(self, n_iter=1, shuffle=False, softmax=False, embedded=True, seed=None):
+    """
+    Creates a base class for Jubatus Classifiers"
+    """
+    self.softmax = softmax
+    self.n_iter = n_iter
+    self.shuffle = shuffle
+    self.embedded = embedded
+    self.seed = seed
+    self.fitted_ = False
+    self.clf_ = None
+
+  @classmethod
+  def _launch_classifier(self):
+    """ 
+    Launch Jubatus Classifier 
+    """
+    raise NotImplementedError()
+
+  def partial_fit(self, X, y):
+    """
+    Partially fit underlying model.
+    If underlying model does not exist, launch a new model.
+    """
+    if getattr(self, 'classes_', None) is None:
```

Comment:
From scikit-learn Contribution pages
```
Attributes that have been estimated from the data must always have a name ending with trailing underscore, for example the coefficients of some regression estimator would be stored in a coef_ attribute after fit has been called.
```
The `classes_` parameter are estimated from the data, and I want to obey their coding style.